Felienne/hedy
# [IMPROVEMENT] Implements dedicated timeout for alert modal

**Description**
Currently the modal timeout is shared between all modals. Due to the re-structuring of the `alert()` modal this is no longer a logical implementation as it is possible for two modals to be shown at the same time. (For example an alert() from the previous interaction and a prompt for the current interaction). In this PR we implement a dedicated timeout for the alert modal to make sure that these don't interfere and close each other at undesired moments.

**Fix for**
This PR fixes #1852.

**How to test**
Find a way to have an alert and any other modal shown at the same time. An easy way to fix this is as follows:

- Invite an invalid user to your class
- Directly after seeing the error message invite another use (stay at the input modal)
- Notice that the input modal does not disappear after the alert does
